lib/view/solve_crossword.dart

```diff
@@ -221,9 +271,100 @@ class TakePictureScreenState extends State<TakePictureScreen> {
     crosswordInfoRepository.insertCrosswordInfo(crosswordInfo);
   }
 
-  void navigateToApp(context) {
-    Navigator.of(context).push(MaterialPageRoute(
-      builder: (context) => const App(),
-    ));
+  SnackBar serverErrorSnackBar(String statusCode) {
```

Comment:
We use this snackbar anywhere?

Reply:
We will. When we will do a refactor, all logs with server errors will be replaced with this snackbar showing server error to the user.